packages/paste-core/components/in-page-navigation/stories/index.stories.tsx

```diff
@@ -0,0 +1,100 @@
+import * as React from 'react';
+import type {Story} from '@storybook/react';
+import {Stack} from '@twilio-paste/stack';
+import {Box} from '@twilio-paste/box';
+import {Heading} from '@twilio-paste/heading';
+import {Tabs, TabList, Tab} from '@twilio-paste/tabs';
+import {CustomizationProvider} from '@twilio-paste/customization';
+import {useTheme} from '@twilio-paste/theme';
+
+import {InPageNavigation, InPageNavigationItem} from '../src';
+
+// eslint-disable-next-line import/no-default-export
+export default {
+  title: 'Components/In Page Navigation',
+  component: InPageNavigation,
+};
+
+export const Default: Story = () => {
+  return (
+    <>
+      <Heading as="h1" variant="heading30">
+        Comparison to Tabs style
+      </Heading>
+      <Stack orientation="vertical" spacing="space120">
+        <Box>
+          This is what <strong>In Page Navigation</strong> looks like (only for visual- clicking should not lead
+          anywhere)
+          <InPageNavigation aria-label="myNav">
```

Comment:
todo (blocking): let's try and look on the web or in product for a real example with a human readable label. Let's use at least 2 different realistic examples on our storybook and website docs.